tests/test_parse_failure_logging.py

```
@@ -0,0 +1,191 @@
+
+
+def _seed_tabs_with_drifted_bubble(parent: str) -> str:
+    ws_root = os.path.join(parent, "workspaceStorage")
+    global_root = os.path.join(parent, "globalStorage")
+    os.makedirs(ws_root, exist_ok=True)
+    os.makedirs(global_root, exist_ok=True)
+
+    ws_dir = os.path.join(ws_root, "ws-a")
+    os.makedirs(ws_dir, exist_ok=True)
+    with open(os.path.join(ws_dir, "workspace.json"), "w", encoding="utf-8") as f:
+        json.dump({"folder": "/tmp/proj"}, f)
```

Ruff (0.15.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/proj"

(S108)